src/components/calculatorbutton.tsx

```diff
@@ -0,0 +1,19 @@
+import { ReactNode } from "react";
+
+interface Props {
+  children: ReactNode;
+  onClick: (value: number | string) => void;
+  label: number | string
+//   className: string;
+}
+
+export default function CalcButton({children, onClick, label}: Props) {
+  return (
+    <button
+      onClick={() => onClick(label)}
+      className="text-center bg-slate-100 active:shadow-sm active:text-sm px-3 py-3 rounded-md flex items-center justify-center shadow-md"
+    >
+      {children}
```

Comment:
This should be replaced with `label` not children.
You don't need the children anymore but maybe you change it in your next iteration.